configuration/uart.md

```diff
@@ -114,31 +110,16 @@ N.B. Selecting the wrong early console can prevent the Pi from booting.
 
 ### Raspberry Pi as terminal
 
-A device you use to login to a serial console is called a ['terminal'](https://en.wikipedia.org/wiki/Computer_terminal). A Raspberry Pi can also act as a terminal, so you can actually login from one Raspberry Pi to another with just a few wires and no network! You just need to connect the two devices' grounds together, and then connect one's TXD to the other's RXD, and vice versa. No resistors or other special hardware are required.
+A Raspberry Pi can also act as a serial terminal, which can be used to connect to another device's serial console. You'll need to disable the terminal's serial console so that the serial console and terminal program won't conflict with each other.
```

Comment:
"terminal's serial console" -> "kernel's serial console" ?

Reply:
I meant "terminal" to refer to the device acting as the terminal, as opposed to the device exposing a serial console. But it was hard to find the right word.

I've revised it in the next commit. Let's see if it's any clearer.